README.md

```diff
@@ -457,6 +458,28 @@ as it's dependant packages.
 
 > 📋 For a sample metapackage, see [MetapackageSample](samples/MetapackageSample/).
 
+### Accesss MSBuild Properties
+
+You may access MSBuild property values of the project being generated for, by first adding the property
+name to the `GeneratorBuildPropertyNames` item group.  For example, if you want to access the TargetFramework build
+property, you would do the following in your generator's .csproj file:
+
+```xml
+<ItemGroup>
+  <GeneratorBuildPropertyNames Include="TargetFramework">
```

Comment:
This is invalid xml. Replace the last character with `/>` to make the xml element self-closing.

Reply:
Oops, good catch.  Replaced this with valid xml in 6deb257